pyiron_lammps/compatibility/calculate.py

```
@@ -0,0 +1,412 @@
+        initial_temperature = 2 * temperature[0]
+
+    if seed <= 0:
+        raise ValueError("Seed must be a positive integer larger than 0")
+
+    thermo_str = ""
+    # Set thermodynamic ensemble
+    if pressure is not None:  # NPT
+        if temperature is None or temperature.min() <= 0:
+            raise ValueError(
+                "Target temperature for fix nvt/npt/nph cannot be 0 or negative"
+            )
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)